dart/lib/src/platform/_io_platform.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:io' as io show Platform;
```

Comment:
the file is called `_io_platform.dart` is a normal pattern to start with `_`?

Reply:
The Flutter Framework does it like this. `_io_platform.dart` and `_web_platform.dart` aren't intended to be imported by any other file than `platform.dart`. It's like a private variable but for files.